scm/native/access.go

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2021 Target Brands, Inc. All rights reserved.
```

Comment:
Could we update the copyright statement to use the current year? 👍 

I think this change will need to be applied throughout the files being added here.